spec/lib/api_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../spec_helper'
+
+RSpec.describe ::Salesforce::Api do
```

Comment:
Not related to this PR but I do not think we have sufficient tests for this class.

Reply:
I think all other things are tested along with the API calls from model files.